src/material/icon/icon.scss

```diff
@@ -45,6 +45,13 @@ mat-icon {
   }
 }
 
+.mat-icon:has(div[style*='letter-spacing:']):has(div[style*='word-spacing:']) {
```

Comment:
Made an update to be targeted similarly to `overflow: visible`, but the text-spacing styles applied by Parashu still seems to be overriding the `.mat-icon` `!important` styles. 

I was wondering if it would be acceptable to hide the icons on mobile and add documentation that the icon will disappear so they will need to make sure the `aria-label` is appropriate even if the icon is missing? Let me know your thoughts @crisbeto.